tests/pipelines/test_pipelines_common.py

```diff
@@ -423,6 +423,53 @@ def test_unbatch_attentions_hidden_states(self):
         self.assertEqual(len(outputs), 20)
 
 
+class PipelineScikitCompatTest(unittest.TestCase):
+
+    def test_pipeline_predict_pt(self):
```

Comment:
You do need to protect them with `@require_torch` and `@require_tf` though.

Reply:
;) yea right. Done .